sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_operations/_patch.py

```diff
@@ -16,13 +15,12 @@
 from azure.core.polling import PollingMethod, LROPoller, NoPolling
 
 from azure.confidentialledger._operations._operations import (
-    ConfidentialLedgerClientOperationsMixin as GeneratedOperationsMixin,
+    _ConfidentialLedgerClientOperationsMixin as GeneratedOperationsMixin,
 )
 from azure.confidentialledger._operations._operations import ClsType, JSON
+import azure.confidentialledger.models as _models
 
-__all__: List[str] = [
-    "ConfidentialLedgerClientOperationsMixin"
-]  # Add all objects you want publicly available to users at this package level
+__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
```

Comment:
```suggestion
__all__: List[str] = ["_ConfidentialLedgerClientOperationsMixin"]  # Add all objects you want publicly available to users at this package level
```
One more suggestion: it is better put customized code before `__all__`